src/excmds.ts

```diff
@@ -4471,6 +4471,9 @@ export function autocmd(event: string, url: string, ...excmd: string[]) {
     // rudimentary run time type checking
     // TODO: Decide on autocmd event names
     if (!AUCMDS.includes(event)) throw new Error(event + " is not a supported event.")
+    if (webrequests.requestEvents.includes(event) && config.get("autocmds", event, url)) {
+        throw new Error(`Autocmd ${event} for ${url} already exist`)
+    }
```

Comment:
I can't merge this because it will break RC files that innocently set exactly the same autocmd every time, so I'll merge without it :)